### ISSUE TITLE
Feature request: Add option to preview whichever file is open on the source pane

### ISSUE THREAD

**Issue by jiwonac** [Enter-tainer/typst-preview#298]
## Feature Request

Suppose you are previewing `foo.typ`. I want to be able to quickly preview `bar.typ` by clicking on the `bar.typ` source code. 

## Use Case

I have a project where I have several typst files that produce separate PDF files. I would like to be able to switch between the different files quickly. This behavior is similar to how Markdown Preview Enhanced works.

**Comment by Enter-tainer** [comment]
I agree this sounds useful in your case but typst-preview works very different from markdown preview. It is much more heavy than that. Namely, each preview panel need a preview server, and each sever launch multiple threads and typst compiler.

Another thing is that not all typst source file is preview-able. So user may see a lot of empty panel when editing these files. This can be unexpected for certain cases
